data/templates/nginx/security.conf.inc

```diff
@@ -33,8 +33,8 @@ more_set_headers "X-Download-Options : noopen";
 more_set_headers "X-Permitted-Cross-Domain-Policies : none";
 more_set_headers "X-Frame-Options : SAMEORIGIN";
 
-# Disable the disaster privacy thing that is FLoC
-more_set_headers "Permissions-Policy : interest-cohort=()";
+# Disable most possible leaking things by default including the horrible FLoC
+more_set_headers "Permissions-Policy : fullscreen=(), geolocation=(), payment=(), accelerometer=(), battery=(), camera=(), microphone=(), magnetometer=(), usb=(), interest-cohort=()";
```

Comment:
Eeeh maybe we can keep allowing camera+microphone, otherwise that may spontaneously break Nextcloud talk or Galene (at least for my work we're using a Galene instance, but i know some people use Nextcloud talk too ?)

I guess the other ones are okay ?

Reply:
Or we modify those headers in Nextcloud talk or Galene? That's a bit the idea of allowing to overwrite them

But yeah, that's the problem that will rise :/